wiki/wiki_entry_template.qmd

```diff
@@ -1,10 +1,30 @@
+Wiki Entry von Matthias Jesse 
+ 
+Key Quote: “Digital health technologies and the disciplinary regimes they configure as part of the practices of self-monitoring and self-care may be said to both empower and disempower patients.”
+ 
+The Key Quote suggests, that Digital Healthcare both has advantages and disadvantages that either makes the patient believe that they feel more autonomous or that they feel obligated to report every health matter. Digital health care technologies have different values for different people, for some people especially older people using these technologies empowers them to live at home for a longer time instead of going into a retirement home. But other people fell that these technologies disempower them because the doctor patient relationship often helps patients to get over the feeling of vulnerability. Especially emotions play a big part in this discussion because only if the patients have trust in these new health technologies, they would use them because when people are ill they are vulnerable. This quote reflects the conflict of these new technologies, the conflict between empowering and disempowering, between autonomy and convenience. Patients will use some of these technologies on some occasions but will decline these on other. This question of using these technologies often can`t be answered because certain situation, require certain measures. Between all aspects we have to keep in mind, that all patients have different demands and needs.
+ 
+The topic of digital healthcare is a topic which is important, but it will become even more important in the future with increasingly improved technology. This article focuses on the decision if digital healthcare is viewed as empowering or disempowering for the patients. This topic is also discussed in other articles with the focus on the key component data. This aspects was included in the text but other articles focuses more on the data security and privacy component. The topic autonomy is an often recurring theme, especially with the elderly generation. Pensioners can continue living at home with technologies that transmit data to their doctors and call emergency if they for example fall down. Algorithms also play a major role, as they can process data better and faster and analyse different behaviours. Ethnic challenges play only a minor role in the edited text, but are becoming more important as technology makes everything more impersonal. I have already mentioned the point of data, another aspect of which is that the data is always available and can always be accessed from anywhere in the world, which safes expenses. This aspect is only barely mentioned. The topic AI or Artificial Intelligence is not mentioned in the text, this is ok because we are in early stages of the AI but this will play a big role in the future not only in digital health care. In general many articles give an overview over this topic rather than focussing on one special topic.            
+ 
```

Comment:
Can you summarise the most important aspects that can harm vs. contribute to autonomy need satisfaction?

Reply:
Digital health leads to more autonomy, as people are no longer tied to one place to always go to the doctor. At the same time, the constant submission of health data leads to being bound to this concept of digital health. If you decide, for example, that you don't want to submit such data on a particular day, the concept no longer works because certain data is missing and the whole picture is jeopardised. The constant reminder to submit data is stressful, depressing and confronting for patients. If one compares the advantages and disadvantages with the background idea of autonomy, then one comes to the conclusion that digital health only provides limited autonomy. Autonomy is the freedom to have one's data checked from anywhere at any time, but at the end of the day a healthcare worker has to check the data and prescribe medicine if necessary. The whole concept has its weak points and will become more and more autonomous over the years; at the moment, only individual groups of people benefit from it and others only in certain situations. the patient is not as autonomous as it first appears due to the constant submission of data and analyses.

Reply:
Thoughts about question number one:

I think that for now, the need for human interaction in healthcare is likely to keep AI on the sidelines as a complement, rather than a substitute, for doctors. But perhaps in a few decades, patients will be comfortable interacting with computers and even trust them as their main source of medical guidance. Some studies suggest that AI can perform such tasks fairly well—and sometimes pick up on signs of disease that doctors miss. But in most cases, the combination of physician expertise and AI was the best option.